docker-compose/dev/apps/config/sampler.json

```diff
@@ -7,5 +7,23 @@
     "port": 7233,
     "namespace": "pocket-ml-testbench",
     "task_queue": "sampler"
-  }
+  },
+  "timeouts": {
+    "random": {
+      "ttft": {
+        "x": [
```

Comment:
"x" and "y" offer no information on what they actually are, use more descriptive labels

Reply:
Fixed, named as "prompt_lenght" and "sla_time" (service level aggrement).